sourcemash/frontend/frontend.py

```diff
@@ -31,7 +31,7 @@ def feeds():
 def feed(id):
 	''' Get articles & pass through to render_template '''
 	feed = Feed.query.get(id)
-	feed_items = Item.query.filter(Item.feed_id==id).all()
+	feed_items = Item.query.filter_by(feed_id=id).all()
```

Comment:
Feed_id is the name of a parameter to filter_by, so it doesn't need to be declared, it just looks up the column in the Item table.

See: http://docs.sqlalchemy.org/en/rel_0_9/orm/query.html#sqlalchemy.orm.query.Query.filter_by